Makefile

```diff
@@ -15,7 +15,10 @@
 # limitations under the License
 #
 
-.PHONY: help clean clean-dist build dev test system-test test-travis release pip-release bin-release dev-binder .binder-image audit audit-licenses
+.PHONY: help clean clean-dist .clean-binder-image .clean-toree-dev-image \
+     build dev test system-test test-travis release pip-release bin-release \
+     dev-binder .binder-image .toree-dev-image \
```

Comment:
`.binder-image` and `.toree-dev-image` should not be listed here as they correspond to files that _are_ the target.

Also, do we need the '.' prefix on `.clean-binder-image` and `.clean-toree-dev-image`?  I think the dotted notation is purely because we don't want the files visible by default.  Because these are PHONY targets, I think removing the dotted prefix is appropriate.

Reply:
regarding the dot, I was just following the same pattern as others